pymtl3/passes/PrintWavePass.py

```diff
@@ -0,0 +1,127 @@
+"""
+========================================================================
+PrintWavePass.py
+========================================================================
+Print the most sigficant number of signals in wave form.
+top object uses top._print_wave(top) to print.
+
+Author : Kaishuo Cheng
+Date   : Oct 4, 2019
+"""
+
+from __future__ import absolute_import, division, print_function
+import time
+from collections import defaultdict
+from copy import deepcopy
+import py
+import sys
+from pymtl3.dsl import Const
+from pymtl3.passes.BasePass import BasePass, PassMetadata
+import random
+
+class PrintWavePass( BasePass ):
+
+  def __call__( self, top ):
+    setattr(top,"_print_wave",_help_print)
```

Comment:
might not need to use setattr here because _print_wave is a constant string

Reply:
I am trying to add a method to the top object, so it can call top._print_wave(top) to print out. 
Is there a better way to do this?

Reply:
You can just do ```top._print_wave = _help_print```.
setattr is usually used when the attribute string is not a constant string

Reply:
Ah, I see. Thanks!